diff_cover/diff_cover_tool.py

```diff
@@ -116,19 +117,28 @@ def parse_coverage_args(argv):
         help=SRC_ROOTS_HELP
     )
 
+    parser.add_argument(
+        '--diff-range-notation',
+        metavar='RANGE_NOTATION',
+        type=str,
+        default='...',
```

Comment:
So wait, I thought there was only double dot and triple dot. Are there more? 

If there only the two perhaps we just have a flag like `--double-dot-diff-notation` or something like that? Im open to thoughts. Mostly just talking it out.

Reply:
Those are the most common. `--double-dot-diff-notation` might be simpler, although it doesn't make it clear what *else* is done if `--double-dot-diff-notation` is passed. On the other hand `--diff-range-notation` being either `...` or `..` makes that more explicit.

Having said that I don't mind changing it. Let me know what you think.

Reply:
You make some good points. I think im ok with it but can you add a small validation function? Even as something as using the `choices` argument to argparse. It would restrict users to `...` and `..` but like... if someone wants another they can open a pr/issue. (If you know of another potentially good one feel free but like... I think starting with what has an immediate need has served the project well to this point) 

Basically I would really want to have an error thrown before some invalid string got sent to a shell.

Reply:
Done! I also rebased on `master`.